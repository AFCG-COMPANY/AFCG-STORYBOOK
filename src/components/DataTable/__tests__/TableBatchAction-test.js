import React from 'react';
import { mount } from 'enzyme';
import { TableBatchAction } from '../';

describe('DataTable.TableBatchAction', () => {
  it('should render', () => {
    const wrapper = mount(<TableBatchAction className="custom-class" />);
    expect(wrapper).toMatchSnapshot();
  });
});
