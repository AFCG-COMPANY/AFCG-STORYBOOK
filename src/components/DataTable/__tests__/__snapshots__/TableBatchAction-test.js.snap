// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableBatchAction should render 1`] = `
<TableBatchAction
  className="custom-class"
  iconDescription="Add"
>
  <Button
    className="custom-class"
    disabled={false}
    icon="add--solid"
    iconDescription="Add"
    kind="ghost"
    small={true}
    tabIndex={0}
    type="button"
  >
    <button
      className="custom-class bx--btn bx--btn--sm bx--btn--ghost"
      disabled={false}
      tabIndex={0}
      type="button"
    >
      <Icon
        className="bx--btn__icon"
        description="Add"
        fillRule="evenodd"
        name="add--solid"
        role="img"
      >
        <svg
          alt="Add"
          aria-label="Add"
          className="bx--btn__icon"
          fillRule="evenodd"
          height="16"
          name="add--solid"
          role="img"
          viewBox="0 0 16 16"
          width="16"
        >
          <title>
            Add
          </title>
          <path
            d="M7 7H4v2h3v3h2V9h3V7H9V4H7v3zm1 9A8 8 0 1 1 8 0a8 8 0 0 1 0 16z"
            key="key0"
          />
        </svg>
      </Icon>
    </button>
  </Button>
</TableBatchAction>
`;
